extras/generate_index.py

```diff
@@ -8,7 +8,7 @@
 
 def main(): 
     # default fallback URL -- just for testing
-    download_url = "https://github.com/CommunityGD32Cores/ArduinoCore-GD32/archive/9fb26ac.zip"
+    download_url = "https://github.com/arcayi/ArduinoCore-GD32/archive/9fb26ac.zip"
```

Comment:
For the merge, this should be reverted to the original address.

Reply:
```suggestion
    download_url = "https://github.com/CommunityGD32Cores/ArduinoCore-GD32/archive/9fb26ac.zip"
```